src/app/globals.css

```diff
@@ -114,3 +114,39 @@
 .addfriend-success{
     @apply mt-1 text-sm text-green-600;
 }
+
+.friend-requests-wrapper{
```

Comment:
out of scope for this document, but intuition tells me the globals.css can be a little more cohesive and less repetitive, this would require looking into the Tailwind docs on another branch